src/scripts/browserDeviceProfile.js

```diff
@@ -660,8 +663,11 @@ export default function (options) {
     }
 
     if (canPlayVp9) {
-        if (!browser.iOS) {
+        if (!browser.iOS && !browser.firefox) {
```

Comment:
Should we check macOS here as Firefox does not really support HDR on any other platforms?

Reply:
Done. I added comments to gate against a regression in case someone wants to enable HDR on Firefox for other platforms before the HDR VP9 bug is fixed.